src/main/kotlin/game/racingcar/strategy/RandomMoveStrategy.kt

```diff
@@ -5,12 +5,13 @@ import kotlin.random.Random
 class RandomMoveStrategy : MoveStrategy {
     override fun move(current: Int): Int {
         return when (Random.nextInt(0, 10)) {
```

Comment:
```suggestion
        return when (1..9.random()) {
```

IntRange를 사용하여 random()을 정의하면 더 간결하지 않을까요?